web/themes/custom/citizen_dart/components/00-base/variables/_general-mixins.scss

```diff
@@ -403,31 +434,108 @@
   box-sizing: border-box;
 }
 
-@mixin arrowLinkBefore($arrowLinkColor){
-  font-family: vinila-extended, sans-serif;
-  font-size: 18px;
-  line-height: 1.44;
+@mixin arrowLinkBefore($arrowLinkColor, $size: lg) {
```

Comment:
Rewrote both arrowLinkBefore and arrowLinkAfter to be a bit more streamlined, and take an extra size specifier, so that both can be called from linkArrow and blockArrow, respectively, and serve basically identical functions, just in either direction.